src/app/components/faq-assistant/faq-assistant.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="container">
-  <div class="wrap">
+  <div [ngClass]="customClass || 'defaultClass'">
```

Comment:
'defaultClass' is redundant, please remove it.

Reply:
I think defaultClass class is necessary, because if developer didn't pass own class then use "defaultClass" class by default

Reply:
No, we create our styles, that user can override or not. But user shouldn't create styles from 0, he should only override.

Reply:
was applied